.codecov.yml

```diff
@@ -0,0 +1,5 @@
+coverage:
+  status:
+    project:
+      default:
+        threshold: 1%
```

Comment:
oh cool! we should add this elsewhere

Reply:
Yeah, it's annoying to get a CI fail for a project drop of 0.01%

Reply:
especially when it's because you just deleted a covered line but patch coverage is 100%!